reactDashboard/src/components/DataCharts/DataCharts.jsx

```diff
@@ -171,18 +181,6 @@ const DataCharts = ({
         borderColor: "DarkGray;",
         borderWidth: 1,
         position: "nearest",
-        callbacks: {
```

Comment:
Why was this removed?

Reply:
I thought this just related to the label for the legend - I will re-insert now that is back

Reply:
Nope, tis to format the numbers on the data point tooltips